src/components/ButtonPanel.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import Button from './Button';
+import '../App.css';
+
+class ButtonPanel extends React.Component {
+  renderSquare(i) {// eslint-disable-line
+    return <Button name={i}/>;// eslint-disable-line
+  }
```

Comment:
- [ ] [MANDATORY] You are not allowed to use eslint-disable. its like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.